mindjunkies/courses/views.py

```
@@ -254,4 +236,4 @@ def form_valid(self, form):
 
         course.update_rating()
 
-        return redirect(reverse("course_details", kwargs={"slug": course.slug}))
+        return redirect(reverse("course_details", kwargs={"slug": course.slug}))
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)